app/models/claims/allocation_filters.rb

```diff
@@ -37,12 +37,12 @@ def lgfs_scopes
     end
 
     def all_fixed_fee
-      where('"claims"."case_type_id" IN (?) OR "claims"."allocation_type" = ?', CaseType.fixed_fee.pluck(:id), 'Fixed')
+      where('"claims"."case_type_id" IN (?) OR "claims"."allocation_type" = ?', CaseType.fixed_fee.select(:id), 'Fixed')
     end
 
     def all_graduated_fees
       where('"claims"."case_type_id" IN (?) OR "claims"."allocation_type" = ?',
-            CaseType.graduated_fees.pluck(:id), 'Grad')
+            CaseType.graduated_fees.select(:id), 'Grad')
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3621/files#r532750493)